.github/workflows/linter.yml

```diff
@@ -0,0 +1,55 @@
+###########################
+###########################
+## Linter GitHub Actions ##
+###########################
+###########################
+name: Lint Code Base
+on: [push, pull_request]
+jobs:
+  super-linter:    
+    name: Apply super-linter
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2      
+      - name: Lint Code Base
+        uses: docker://github/super-linter:v2.1.1
+        env:
+          VALIDATE_ALL_CODEBASE: true
+          VALIDATE_PYTHON: true
+          VALIDATE_BASH: true
+          VALIDATE_MD: true
+          VALIDATE_YAML: true
+          VALIDATE_XML: true
+          DISABLE_ERRORS: true # TODO: Eventually this will be removed
```

Comment:
I'd suggest to remove the comment and create a GitHub issue for this.

Reply:
I think it is best to create the issue and mantain the comment to help to locate were the change is needed.